app/helpers/setCookie.js

```diff
@@ -0,0 +1,18 @@
+import { helper } from '@ember/component/helper';
+
+function setCookie(mode) {
+  const domain = '.realdevsquad.com';
+  const cookieExpiryDate = new Date(Date.now() + 24 * 31 * 60 * 60 * 1000);
+  const cookie = `theme=${mode}; domain=${domain}; expires=${cookieExpiryDate} path=/`;
+  document.cookie = cookie;
+
+  if (mode === 'light') {
+    document.body.classList.remove('darkTheme');
+    document.body.classList.add('lightTheme');
+  } else {
+    document.body.classList.remove('lightTheme');
+    document.body.classList.add('darkTheme');
```

Comment:
@akshay1502 could we use `classList.toggle` here? https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle

Reply:
I tried using the toggle. But in js, we can't toggle two classes like one adding and one removing, both will be removed or added. Whereas Jquery supports adding and removing two classes using the toggle.

Reply:
Use two toggle statements?

Reply:
@swarajpure i tried but it doesn't work properly. So I stuck to my method.